docs/generate_shipping_label.rst

```diff
@@ -0,0 +1,35 @@
+Generate Shipping Label
+========================
+
+Shipping labels help in differentiating between different purchase orders.
+
+This documentation explains how shipping label can be generated for all
+shipments.
+
+.. note::
+
+   Shipping label can only be generated if the order is in the confirmed 
+   state. To create a new shipment, refer `Creating a New Shipment
+   <file:///home/shaambhavi/devel/trytond-shipping-ups/trytond-shipping-ups/build/create_new_shipment.html>`_.
```

Comment:
Link is from your local file, it wont work. And this is not the right way to interlink files within a doc.